boskos/cmd/mason/main.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors
```

Comment:
change 2017 to 2018 here and elsewhere for new files.

Reply:
done